content/docs/getting_started/upgrade.md

```diff
@@ -103,6 +103,44 @@ Omit the `--values` flag if you prefer to use the default settings.
 
 Run `helm upgrade --help` for more options.
 
+## Upgrading Third Party Dependencies
+
+### Envoy
+
+The envoy version can be updated by changing the value of the `envoyImage` variable in the osm-mesh-config. For instance, to update the envoy-alpine image to v1.19.1, the following command should be run: 
+
+```bash
+kubectl patch meshconfig osm-mesh-config -n osm-system -p '{"spec":{"sidecar":{"envoyImage":"envoyproxy/envoy-alpine:v1.19.1"}}}' --type=merge
+```
+
+After the meshconfig has been updated, all the services and deployments that are part of the mesh must be restarted so they can pull the updated envoy image. For instance, if the user is running the [manual demo](quickstart/manual_demo.md), then they would restart the following deployments: 
+
+```console
+  kubectl rollout restart deploy bookbuyer -n bookbuyer
+  kubectl rollout restart deploy bookstore -n bookstore
+  kubectl rollout restart deploy bookthief -n bookthief
+  kubectl rollout restart deploy bookwarehouse -n bookwarehouse
+```
```

Comment:
```suggestion
After the MeshConfig resource has been updated, all the pods and deployments that are part of the mesh must be restarted so that the newer version of Envoy sidecar can be injected onto the pods as a part of the automatic sidecar injection that OSM performs.